airbyte-integrations/base-python/Dockerfile

```diff
@@ -2,14 +2,19 @@ FROM airbyte/base:dev
 COPY --from=python:3.7-slim / /
```

Comment:
We should make sure that this is the right way to do it.

Reply:
Oh sorry, forgot about the previous comment. I'll also fix `eval` usage.

Reply:
I think that this method of copying files, but after looking closer it doesn't look like this is an appropriate way of running Python, since env vars aren't copied over https://github.com/docker-library/python/blob/master/3.7/buster/slim/Dockerfile

In https://github.com/airbytehq/airbyte/pull/490/commits/e6c28985e075e051504dee51cf06338118eaeba2 I switch to copying files and re-declaring the entrypoint. This isn't ideal, since we have to be careful not to introduce ENV var or files outside of /airbyte in `airbyte/base:dev`.